mono/metadata/mbedtls.c

```diff
@@ -0,0 +1,289 @@
+#ifdef HAVE_CONFIG_H
```

Comment:
I think we should name these two files `mono-mbedtls.h/c` to avoid potential conflicts.